lib/jnpr/junos/utils/start_shell.py

```diff
@@ -21,28 +22,39 @@ def _ssh_exec(self, command):
 
     """
 
-    def __init__(self, nc):
+    def __init__(self, nc, timeout=30):
```

Comment:
Should `timeout` be a property of the `StartShell` object (as you've done here), or should it come from the `timeout` property of the `Device` object (`nc`), or we could do both by changing this to:
`def __init__(self, nc, timeout=0):`
and my comment on line 35.